pkg/virt-launcher/virtwrap/api/converter.go

```diff
@@ -985,6 +987,24 @@ func Convert_v1_VirtualMachine_To_api_Domain(vmi *v1.VirtualMachineInstance, dom
 		}
 	}
 
+	// Append HostDevices to DomXML if GPU is requested
+	if util.IsGpuVmi(vmi) {
+		vgpuMdevUUID := append([]string{}, c.VgpuDevices...)
+		hostDevices, err := createHostDevicesFromMdevUuidList(vgpuMdevUUID)
+		if err != nil {
+			log.Log.Reason(err).Error("Unable to parse Mdev Uuid addresses")
+		} else {
+			domain.Spec.Devices.HostDevices = append(domain.Spec.Devices.HostDevices, hostDevices...)
+		}
+		gpuPciAddresses := append([]string{}, c.GpuDevices...)
+		hostDevices, err = createHostDevicesFromPCIAddresses(gpuPciAddresses)
+		if err != nil {
+			log.Log.Reason(err).Error("Unable to parse pci addresses")
```

Comment:
(if you respin anyway because of other comments then) please change pci -> PCI

Reply:
Updated